### PR TITLE
docs: minor fixes across docs

### PR DESCRIPTION
**Pull request by gmastrapas**

### CODE REVIEW

File: docs/basics/datasets/session-dataset.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 (session-dataset)=
 # Session Dataset
 
-A {class}`~fintuner.tuner.dataset.SessionDataset` is a `DocumentArray`, in which each Document has `matches` Document. The labels are stored under matched Document's `.tags['finetuner_label']`. The label can be either `1` (denoting similarity of match to its parent `Document`) or `-1` (denoting dissimilarity of the match from its parent `Document`).
+A {class}`~fintuner.tuner.dataset.SessionDataset` is a `DocumentArray`, in which each Document has `matches` Documents. The labels are stored under matched Document's `.tags['finetuner_label']`. The label can be either `1` (denoting similarity of match to its parent `Document`) or `-1` (denoting dissimilarity of the match from its parent `Document`).
```
Comment (tadejsv) [inline docs/basics/datasets/session-dataset.md:4]: I would put just `matches`

File: finetuner/labeler/executor.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import abc
-from typing import Dict, Optional
+import numpy as np
 
 from jina import Executor, DocumentArray, requests, DocumentArrayMemmap
 from jina.helper import cached_property
+from typing import Dict, Optional
```
Comment (bwanglzu) [inline finetuner/labeler/executor.py:6]: can you fix the import order?
```python
# python built-in
import abc
from typing .. 
# 3rd party stuff
import numpy as np
from jina ...
# finetuner stuff
from ..embedding import embed
...
```

**Review by tadejsv — APPROVED** [review]
Looks good

File: docs/basics/fit.md [hunk 3]
```diff
@@ -55,62 +55,83 @@ Note that, `model` above must be an {term}`Embedding model`.
...
 
 ```python
+import finetuner
 import torch
+
+from collections import Counter
 from finetuner.toydata import generate_qa
-import finetuner
+from jina import Document
+from typing import List
```
Comment (bwanglzu) [inline docs/basics/fit.md:64]: same, ordered imports.

**Review by bwanglzu — CHANGES_REQUESTED** [review]
minor comments, you can use [isort](https://github.com/PyCQA/isort)

**Comment by bwanglzu** [comment]
please resolve the conflicts. I rewrite tailor docs yesterday

File: docs/components/tuner.md [hunk 4]
```diff
@@ -27,27 +27,27 @@ Loss function of the Tuner can be specified via the `loss` argument of `finetune
...
+$$\ell_{i, p, n}=\max(0, d(\mathbf{x}_i, \mathbf{x}_p)-d(\mathbf{x}_i, \mathbf{x}_n)+m)$$, where Document $p$ and $i$ are positively related, whereas $n$ and $i$ are negatively related or unrelated, $d(\cdot, \cdot)$ representes a distance function, and $m$ is the desired distance (wedge) between the positive and negative pairs.
 :::
 
 
 ```{tip}
 
-Although siamese and triplet loss works on pair and triplet inputs respectively, there is **no need** to worry about the data input format. You only need to make sure your data is labeled according to {ref}`data-format`, then you can switch between all losses freely.
+Although siamese and triplet loss work on pair and triplet inputs respectively, there is **no need** to worry about the data input format. You only need to make sure your data is labeled according to {ref}`data-format`, then you can switch between all losses freely.
```
Comment (gvondulong) [inline docs/components/tuner.md:50]: there should be a comma between `inputs` and `respectively`.

File: docs/get-started/celeba.md [hunk 5]
```diff
@@ -4,27 +4,27 @@
...
 ```{caution}
 Beware that the original CelebA dataset is 1.3GB. In this example, we do not need the full dataset. Here is a smaller version which contains 1000 images from the original dataset. You can [download it from here](https://static.jina.ai/celeba/celeba-img.zip).
 ```
 
-Note that Finetuner accepts Jina `DocumentArray`/`DocumentArrayMemmap`, so we first load CelebA data into this format using generator:
+Note that Finetuner accepts Jina `DocumentArray`/`DocumentArrayMemmap`, so we first load CelebA data into this format using a generator:
 
 ```python
-from jina.types.document.generators import from_files
+from docarray.document.generators import from_files
```
Comment (gvondulong) [inline docs/get-started/celeba.md:27]: I think this will not work with older versions of Jina. 
I am not sure if we should import the document.generators into jina.__init__.py
Then we could do: `from jina import from_files`
Maybe talk to someone from the core team such as Tobias or Joan I am not too sure about it.

File: docs/get-started/celeba.md [hunk 6]
```diff
@@ -39,7 +39,7 @@ for doc in data:
 
 ## Load the pretrained model
 
-Let's import pretrained ResNet50 as our base model. ResNet50 is implemented in PyTorch, Keras and Paddle. You can choose whatever framework you feel comfortable:
+Let's import a pretrained ResNet50 as our base model. ResNet50 is implemented in PyTorch, Keras and Paddle. You can choose the framework you feel more comfortable with:
```
Comment (gvondulong) [inline docs/get-started/celeba.md:42]: chose framework you are the most comfortable with
or: 
chose whichever framework you are more comfortable with

**Review by bwanglzu — APPROVED** [review]
LGTM!

File: docs/get-started/swiss-roll/index.md [hunk 7]
```diff
@@ -133,14 +133,11 @@ And finally call Finetuner:
 
 ```python
 import finetuner
-new_model, summary = finetuner.fit(embed_model, train_data=da, epochs=500)
+new_model = finetuner.fit(embed_model, train_data=da, epochs=500)
```
Comment (gvondulong) [inline docs/get-started/swiss-roll/index.md:136]: 👍

**Review by gvondulong — COMMENTED** [review]
Great job! Thanks for going through these! 😻

**Comment by bwanglzu** [comment]
we need to create a ticket that  pure docs change won't trigger unit/integration test
